### PR TITLE
add BoardTest, Update test

### PR DESCRIPTION
**Pull request by dulphyj**
Add test for board class, for update function, won or lose text reads

### CODE REVIEW

File: src/main/java/org/fundacionjala/at15/spaceinvaders/Constants.java [hunk 1]
```diff
@@ -17,6 +17,7 @@
         static final int START_Y = 550;
         static final int VELOCITY_X = 8;
         static final int DIAMETER = 15;
+        static final int DEATHE = 45;
```
Comment (adhes14) [inline src/main/java/org/fundacionjala/at15/spaceinvaders/Constants.java]: You should name it DEATH

**Review by adhes14 — CHANGES_REQUESTED** [review]
Please see my comment in the code

**Review by adrianovi — COMMENTED** [review]
You should use this:
protected void setDeaths(int parameter) {
        this.deaths = parameter;
    }

for set the deaths

**Review by adhes14 — APPROVED** [review]

**Comment by adhes14** [comment]
Well done!

**Pull request merged**